src/VerifyExamples/ExposedApi.elm

```diff
@@ -0,0 +1,68 @@
+module VerifyExamples.ExposedApi
+    exposing
+        ( ExposedApi
+        , definitions
+        , filter
+        , parse
+        )
+
+import Regex exposing (HowMany(..), Regex)
+import Regex.Util exposing (firstSubmatch, newline)
+
+
+type ExposedApi
+    = Explicitly (List String)
+    | None
+
+
+parse : String -> List String -> ExposedApi
+parse value functions =
+    value
+        |> firstSubmatch moduleExposing
+        |> Maybe.map (toExposedDefinition functions)
+        |> Maybe.withDefault None
+
+
+toExposedDefinition : List String -> String -> ExposedApi
+toExposedDefinition functions match =
+    case match of
+        ".." ->
```

Comment:
we are missing aliases and types in the case where a moudle exposes everything. maybe we should just change the warning in this case that a module should be explicit about it's exposed definitions.

Reply:
👍